commons-app/pom.xml

```diff
@@ -34,6 +30,15 @@
 			<name>SpringSource Enterprise Bundle Repository - SpringSource Bundle Releases</name>
 			<url>http://repository.springsource.com/maven/bundles/release</url>
 		</repository>
+
+		<repository>
+			<id>3rdParty</id>
```

Comment:
This is for receiving the ms sql jdbc driver. Have you checked if this resolves from outside, i.e., when not on the SWC network?